pkg/plugins/backendplugin/instrumentation/instrumentation.go

```diff
@@ -29,6 +29,13 @@ var (
 		Help:      "Plugin request duration",
 		Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10, 25, 50, 100},
 	}, []string{"plugin_id", "endpoint", "target"})
+
+	PluginRequestDurationSeconds = prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
Nit: This can be unexported

Reply:
I plan to import the metric and use it in the api endpoint `/api/frontend-metric`